iconloaderlib/build.gradle

```diff
@@ -11,9 +11,10 @@ android {
             res.srcDirs = ['res']
         }
     }
-    addFrameworkJar('android.jar')
 }
 
+addFrameworkJar('framework-13.jar')
+
 dependencies {
     implementation "androidx.palette:palette-ktx:1.0.0"
     compileOnly fileTree(dir: FRAMEWORK_PREBUILTS_DIR, include: 'framework-12l.jar')
```

Comment:
@MrSluffy Seems we should remove this?

Reply:
No, keep it for the mean time, I'll start working on QS support a12 to a10 in 13-dev branch in the end of the month. We can remove it once we check and not needed anymore

Reply:
Gotcha.